### PR TITLE
Note that Folder Collections require 'title' field

### PR DESCRIPTION
**Pull request by remmah**
**- Summary**

Updated docs so people know that the title field is required for Folder collections

**- Test plan**

Not sure why the last line was added and removed...

**- Description for the changelog**

Updated docs to note the requirement of a title field for creating new Folder Collection entries

**- A picture of a cute animal (not mandatory but encouraged)**

### CODE REVIEW

**Comment by verythorough** [comment]
Deploy preview for *netlify-cms-www* ready!

Built with commit 033757349e98c4ef82f9dd2545b5f2e7d50f9c00

https://deploy-preview-1037--netlify-cms-www.netlify.com

**Comment by verythorough** [comment]
Deploy preview for *cms-demo* ready!

Built with commit 033757349e98c4ef82f9dd2545b5f2e7d50f9c00

https://deploy-preview-1037--cms-demo.netlify.com

File: website/site/content/docs/collection-types.md [hunk 1]
```diff
@@ -30,6 +30,8 @@ Example:
     - {label: "Body", name: "body", widget: "markdown"}
 ```
 
+**Note:** Folder Collections currently require a `title` field for creating new entries.
```
Comment (erquhart) [inline website/site/content/docs/collection-types.md]: I'd say put this right before "Example:", and add a bit more so it's clear what we mean by "title field":

> Folder collections must have at least one field with the name "title" for creating new entry slugs. That field should use the default "string" widget. The "label" for the field can be any string value.

**Comment by erquhart** [comment]
Thanks for this @remmah! Change request for you, and then this should be good to merge.

**Comment by remmah** [comment]
Change request implemented!

**Comment by erquhart** [comment]
Awesome - and thanks for adding yourself to the all-contribs :)

**Comment by dirtyredz** [comment]
took me about 2 hours to figure that one out :)

**Pull request merged**